cuebot/src/test/java/com/imageworks/spcue/test/dispatcher/HostReportHandlerTests.java

```diff
@@ -297,14 +297,16 @@ public void testHandleHostReportWithFullTemporaryDirectories() {
         *   Precondition:
         *     - HardwareState=UP
         *   Action:
-        *     - Receives a HostReport with freeTempDir < dispatcher.min_bookable_free_temp_dir_kb (opencue.properties)
+        *     - Receives a HostReport with less freeTempDir than the threashold 
+        *       (opencue.properties: min_available_temp_storage_percentage)
         *   Postcondition:
         *     - Host hardwareState changes to REPAIR
-        *     - A comment is created with subject=SUBJECT_COMMENT_FULL_TEMP_DIR and user=CUEBOT_COMMENT_USER
+        *     - A comment is created with subject=SUBJECT_COMMENT_FULL_TEMP_DIR and 
+        *       user=CUEBOT_COMMENT_USER
         * */
-        // Create HostReport
+        // Create HostReport with totalMcp=4GB and freeMcp=128MB
```

Comment:
"Don't repeat yourself" (DRY). Remove `with totalMcp=4GB and freeMcp=128MB`. It might change in the future.

Reply:
I agree putting values on comments is not a good practice, but for a unit test like this, where part of the informations is hidden on the function (`getRenderHostBuilder`) it is useful to have the values we're comparing handy to understand the logic. I'd say for unit test, legibility wins over DRY. Don't you agree?